### ISSUE TITLE
Mobile sidebar header layout

### ISSUE THREAD

**Issue by Andreas-Kainz** [CollaboraOnline/online#6861]
I suggest to make the sidebar header layout more similar to the regular sidebar header layout. Maybe we can share css vars.

| Mobile | Desktop |
| ----------- | ------------- |
| ![Screenshot_20230709-150912_1.png](https://github.com/CollaboraOnline/online/assets/8517736/e0d8c411-81c8-4d9d-affb-bdaed2f3125e) | ![CollaboraOnline_220510_Writer_Zotero_CitationPreference_FirstCitation_edit_CP-1320x743_1.png](https://github.com/CollaboraOnline/online/assets/8517736/aa486146-9f28-43a2-a3fc-d13df6854c97) |

**Comment by Andreas-Kainz** [comment]
@pedropintosilva any feedback about mobile sidebar layout for the headers at the example for spacing:

**Comment by Andreas-Kainz** [comment]
@pedropintosilva any feedback about mobile sidebar layout for the headers at the example for spacing:

**Comment by pedropintosilva** [comment]
Thanks for reporting @Andreas-Kainz 

# Context
## Bottom Line
In the past we set the headings to include a line under them to help users spot where the sections end and start (specially useful when scrolling quickly up and down within mobile wizard

## Problems
We were getting some headings that were not needed (we already have that info repeated in the label of the commands etc) and so we ended up hiding  some of them (with display: none)

The thing is now we have more and probably we should double check if there is any other in need to be hidden

# Layout
> I suggest to make the sidebar header layout more similar to the regular sidebar header layout. 

I agree we should improve those. But I don't know what you have in mind ^. 
I was thinking that we should simplify:
* Use similar gray header as in the sidebar (sized adapted to mobile). After all this is just a non-interactive element
* Remove the bottom line
* Add divider in between sections (so, above the header) 
 basically something similar to what people expect (see android's navigation drawer headers with dividers https://m3.material.io/components/navigation-drawer/overview )

**Comment by Andreas-Kainz** [comment]
I will make additional changes at the mobile "sidebar" layout, but step by step and with issues where we can discuss.
